snippets/snippets.cson

```diff
@@ -1,119 +1,118 @@
 # Generated with Atomizr - https://github.com/idleberg/atomizr.rb
-'.source.js,source.jsx':
-  'componentWillMount()':
-    'prefix': 'cwm'
-    'body': """
-      componentWillMount() {
-        ${1}
-      }$0
-    """
+'.source.jssource.jsx':
```

Comment:
add a comma here so it looks like `.source.js,source.jsx`

Reply:
Will update, thanks